frontend/components/icons/stars-icon.tsx

```
@@ -0,0 +1,28 @@
+const StarsIcon = () => {
+    return (
+        <svg width='49' height='37' viewBox='0 0 49 37' fill='none' xmlns='http://www.w3.org/2000/svg'>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)